test/e2e/search/protractor.conf.js

```diff
@@ -14,6 +14,6 @@ exports.config = {
   },
   // If ng-app attribute is in a descendant of <body>, tell Protractor where ng-app is
   rootElement: '#main-content',
-  // PTOR_BASE_URL should be http://dev.facebase.org for now.
-  baseUrl: process.env.CHAISE_BASE_URL + '/data/search'
+  // CHAISE_BASE_URL should be http://dev.misd.isi.edu/chaise for now.
+  baseUrl: 'http://dev.misd.isi.edu/chaise' + '/search'
```

Comment:
Ok, I changed it to 
```
baseUrl: process.env.CHAISE_BASE_URL + '/search'
```
I remember Jessie told me this is the preferred way, because later '/record' and '/login' may be concatenated to the CHAISE_BASE_URL. 

Do you think it's alright?